pyproject.toml

```diff
@@ -193,21 +179,22 @@ envlist = sqlalchemy{ 1.3, 1.4, 2.0, 2.0-sqlmodel }-{ asyncpg, noasyncpg }-{aws_
 
 [testenv]
 passenv = CI
```

Comment:
is this still necessary?

Reply:
ah yeah, I think it's used in the pytest plugin

Reply:
Yes, the CI injects the postgres variables that are needed to connect to the DB.

Reply:
Yeah [this](https://github.com/dialoguemd/fastapi-sqla/blob/master/fastapi_sqla/_pytest_plugin.py#L31)